.github/workflows/ci.yml

```
@@ -14,14 +14,19 @@ jobs:
         node-version: [12.x, 13.x, 14.x, 15.x, 16.x, 17.x, 18.x, 19.x, 20.x, 21.x, 22.x]
     steps:
       - uses: actions/checkout@v3
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js 22.x To Build
```

suggestion: let's drop the version here as it'll inevitably get missed in a future update when the actual node version below gets bumped but the title gets forgotten.
```suggestion
      - name: Use Node.js To Build
```